packages/gravity-ui-web/src/sass/01-tools/_layouts.scss

```diff
@@ -0,0 +1,53 @@
+// a mixin to add column based layout styles
+@mixin grav-l-column-layout($column-count: 2, $column-spacing: $grav-sp-m) {
```

Comment:
Hmm.. I think the name `$column-spacing` is slightly ambiguous. I this case it is _half_ the gutter width, but without looking at the code I probably would have assumed this parameter was meant to be the _whole_ gutter, or "gap" if you prefer, width - i.e. the spacing between columns.

When talking about layout grids, "gutter" is a commonly used and fairly well understood term meaning the whole gap between columns (or rows): https://uxdesign.cc/the-anatomy-of-a-grid-c955d5355fae

However, a brief chat with @danbull reminded me that in CSS they tend to use the word "gap" for same purpose. E.g. `column-gap`, `grid-column-gap`, etc. so maybe that's a better name to use rather than "gutter".

So perhaps something along these lines would be more intuitive:

```scss
@mixin grav-l-column-layout($column-count: 2, $column-gap: $grav-sp-l) {
  $column-spacing: $column-gap / 2;
  ...
}
```